h2o-docs/src/product/getting-started/getting-started.rst

```diff
@@ -0,0 +1,194 @@
+
+
+Algorithms
+----------
+
+`This section describes the science behind our algorithms <../data-science.html#data-science>`__ and provides a detailed, per-algorithm view of each model type.
+
+Use cases
+---------
+
+H2O can handle a wide variety of practical use cases due to its robust catalogue of supported algorithms, wrappers, and machine learning tools. The following are some example problems H2O can handle:
+
+- Determining outliers in housing prices based on number of bedrooms, number of bathrooms, access to waterfront, etc. through `anomaly detection <https://github.com/h2oai/h2o-tutorials/tree/master/best-practices/anomaly-detection>`__.
+- Revealing natural customer `segments <https://github.com/h2oai/h2o-tutorials/tree/master/best-practices/segmentation>`__ in retail data to determine which groups are purchasing which products.
+- Linking multiple records to the same person with `probabilistic matching <https://github.com/h2oai/h2o-tutorials/tree/master/best-practices/probabilistic-matching-engine>`__.
+- Unsampling the minority class for credit card fraud data to handle `imbalanced data <https://github.com/h2oai/h2o-tutorials/tree/master/best-practices/imbalanced-data>`__. 
+- `Detecting drift <https://github.com/h2oai/h2o-tutorials/tree/master/best-practices/drift-detection>`__ on avocado sales pre-2018 and 2018+ to determine if a model is still relevant for new data.
+
+See our `best practice tutorials <https://github.com/h2oai/h2o-tutorials/tree/master/best-practices>`__ to further explore the capabilities of H2O.
+
+New user quickstart
+-------------------
+
+You can follow these steps to quickly get up and running with H2O directly from the `H2O-3 repository <https://github.com/h2oai/h2o-3>`__. These steps will guide you through cloning the repository, starting H2O, and importing a dataset. Once you're up and running, you'll be better able to follow examples included within this user guide.
+
+1. In a terminal window, create a folder for the H2O repository:
+
+.. code-block:: bash
+
+   user$ mkdir ~/Desktop/repos
+
+2. Change directories to that new folder, and then clone the repository. Notice that the prompt changes when you change directories:
+
+.. code-block:: bash
+
+    user$ cd ~/Desktop/repos
+    repos user$ git clone https://github.com/h2oai/h2o-3.git
+
+3. After the repository is cloned, change directories to the ``h2o-3`` folder:
+
+.. code-block:: bash
+
+    repos user$ cd h2o-3
+    h2o-3 user$
+
+4. Run the following command to retrieve sample datasets. These datasets are used throughout the user guide and within the `booklets <../additional-resources.html#algorithms>`__.
+
+.. code-block:: bash
+
+   h2o-3 user$ ./gradlew syncSmalldata
+
+At this point, choose whether you want to complete this quickstart in Python or R. Then, run the following corresponding commands from either the Python or R tab:
+
+.. tabs::
```

Comment:
Just a tiny note: Did we finally decide that the Python tab should be the first in the tab? So maybe we should change this everywhere (in another PR). :)

Reply:
That is the plan to flip everywhere it around as I go through!